lib/Swagger/src/TemplateClassification/Storage/Api/TCSApi.php

```diff
@@ -165,71 +169,65 @@ public function getTemplateDetailsWithHttpInfo($wiki_id, $page_id)
         $resourcePath = str_replace("{format}", "json", $resourcePath);
 
         
-        
-  
         // for model (json/xml)
         if (isset($_tempBody)) {
             $httpBody = $_tempBody; // $_tempBody is the method argument, if present
         } elseif (count($formParams) > 0) {
             $httpBody = $formParams; // for HTTP post (form)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)